rules/otp.bzl

```diff
@@ -137,6 +137,8 @@ otp_alert_digest = rule(
         "_opentitantool": attr.label(
             default = "//sw/host/opentitantool:opentitantool",
             allow_single_file = True,
+            executable = True,
```

Comment:
Above, on line 118, we should have `"--rcfile="` as well to prevent the opentitantool command from parsing any user's local configuration.

That's probably not causing any error in CI right now, but it may cause an issue if/when people execute the rule on their local machines.

Reply:
Done.